### ISSUE TITLE
Check does not work for Tenx input parameters.

### ISSUE THREAD

**Issue by grasshoffm** [caleblareau/mgatk#52]
**Describe the bug**
I set up the mgatk parameters to run in the tenx mode. 

I used to check to control my input parameters and I got this error.

```
Mon Dec 13 11:05:30 CET 2021: mgatk v0.6.4
Mon Dec 13 11:05:30 CET 2021: checking dependencies...
Traceback (most recent call last):
  File "/home/mg000001/miniconda3/envs/Maegtk_Python38_V_0_6_4/bin/mgatk", line 8, in <module>
    sys.exit(main())
  File "/cm/shared/apps/scRNA/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/cm/shared/apps/scRNA/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/cm/shared/apps/scRNA/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/cm/shared/apps/scRNA/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/mg000001/miniconda3/envs/Maegtk_Python38_V_0_6_4/lib/python3.8/site-packages/mgatk/cli.py", line 232, in main
    if bams[0] == '':
IndexError: list index out of range
```

Tenx requires a bam file as input, while call requires a folder. The check assumes that you have supplied a folder and will always fail for a tenx setup.

**Comment by caleblareau** [comment]
thanks for the note; `check` was added before `tenx` and never updated. I'll flag it here and update the code.

**Comment by iiam-health-official** [comment]
I have encountered an identical issue while running v0.6.4 of the code. I was wondering if there are modifications I can make to the code to address the bug? Thank you.

**Comment by caleblareau** [comment]
I haven't yet fixed this but hopefully it should just work with `tenx` and you can skip the `check` for now maybe by using a smaller barcode list (e.g. only 5 cells) to quickly prototype the run?

**Comment by caleblareau** [comment]
issue should now be solved in version 0.6.6, going live on Pypi tonight. Thanks for raising this.
